resources/src/jvmMain/kotlin/dev/icerock/moko/resources/FileResource.kt

```diff
@@ -11,8 +11,10 @@ actual open class FileResource(
     val filePath: String
 ) {
     fun readText(): String {
-        val stream = resourcesClassLoader.getResourceAsStream(filePath)
-            ?: throw FileNotFoundException("Couldn't open resource as stream at: $filePath")
+        // https://github.com/icerockdev/moko-resources/issues/424
+        val jvmFilePath = filePath.replace('\\', '/')
```

Comment:
as i see in comment of issue - better to replace `File.separatorChar` to `/` in generator. we not have any reasons to use `File.separatorChar`, yes?

Reply:
Yeah , I didn't want to mess with the generator because I didn't know what other stuff would go wrong because of a slash change